### PR TITLE
README.md update

### PR DESCRIPTION
**Pull request by FK78**
Resolves #1846 

**Overall change:** _Update README.md_

**Code changes:**

- _Added desc for exported functions._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by thekp — APPROVED** [review]

**Review by oluoluoxenfree — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Review by thekp — APPROVED** [review]
🎉

File: package.json [hunk 1]
```diff
@@ -105,8 +105,8 @@
     "lint-staged": "^9.2.2",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.9.0",
```
Comment (dr3) [inline package.json]: We need to change these all at once, with the packages. Can you undo these please?

File: package.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
     "@bbc/psammead-story-promo": "2.3.3",
     "@bbc/psammead-storybook-helpers": "^5.0.0",
     "@bbc/psammead-styles": "^2.0.3",
-    "@bbc/psammead-test-helpers": "^1.1.1",
+    "@bbc/psammead-test-helpers": "^1.3.0",
```
Comment (dr3) [inline package.json:72]: This will need a changelog change and base level package bump :)

**Review by dr3 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.5.3 | [PR#1849](https://github.com/bbc/psammead/pull/1849) Update README.md |
```
Comment (dr3) [inline CHANGELOG.md]:
```suggestion
| 1.5.3 | [PR#1849](https://github.com/bbc/psammead/pull/1849) Bump @bbc/psammead-test-helpers `^1.1.1` to `^1.3.1`  |
```

**Comment by thekp** [comment]
@dr3 we had to bump the base package to `1.3.0` because `1.3.1` has not been published yet.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.5.3 | [PR#1849](https://github.com/bbc/psammead/pull/1849) Bump @bbc/psammead-test-helpers `^1.1.1` to `^1.3.1`  |
```
Comment (dr3) [inline CHANGELOG.md]:
```suggestion
| 1.5.3 | [PR#1849](https://github.com/bbc/psammead/pull/1849) Bump @bbc/psammead-test-helpers `^1.1.1` to `^1.3.0`  |
```

File: packages/utilities/psammead-test-helpers/README.md [hunk 5]
```diff
@@ -11,8 +11,8 @@ This package provides a collection of helper methods for implementing Jest snaps
...
-| setWindowValue             | key, value                                  | {}                                                                                                                                                                                                                               |
-| resetWindowValue           | key, value                                  | {}                                                                                                                                                                                                                               |
+| setWindowValue             | key, value                                  | {Allows you to set variables on the window (eg. location) that are normally not writable}                                                                                                                                        |
```
Comment (dr3) [inline packages/utilities/psammead-test-helpers/README.md]:
```suggestion
| setWindowValue             | key, value                                  | Allows you to set variables on the window (eg. location) that are normally not writable                                                                                                                                        |
```

File: packages/utilities/psammead-test-helpers/README.md [hunk 6]
```diff
@@ -11,8 +11,8 @@ This package provides a collection of helper methods for implementing Jest snaps
...
+| resetWindowValue           | key, value                                  | {Allows you to reset the WindowValues that are previously overwritten by the setWindowValue function}                                                                                                                            |
```
Comment (dr3) [inline packages/utilities/psammead-test-helpers/README.md]:
```suggestion
| resetWindowValue           | key, value                                  | Allows you to reset the WindowValues that are previously overwritten by the setWindowValue function                                                                                                                           |
```

**Review by dr3 — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Pull request merged**